k8s/edge/Dockerfile

```diff
@@ -13,40 +13,38 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 #
-FROM openresty/openresty:alpine
+FROM openresty/openresty:1.13.6.2-alpine
 
-# Dumb-init
 ENV DUMB_INIT_VERSION 1.2.0
+
+# Dumb-init, python3, lua
 RUN set -ex \
   && apk add --no-cache --virtual .install-deps openssl \
+       python3 python3-dev bash g++ zlib zlib-dev jpeg jpeg-dev \
+       ca-certificates gnupg openssl git curl luarocks5.3 \
+  && luarocks-5.3 install lua-resty-statsd 3.0.3-1 \
   && wget "https://github.com/Yelp/dumb-init/releases/download/v${DUMB_INIT_VERSION}/dumb-init_${DUMB_INIT_VERSION}_amd64" \
   && wget "https://github.com/Yelp/dumb-init/releases/download/v${DUMB_INIT_VERSION}/sha256sums" \
   && grep "dumb-init_${DUMB_INIT_VERSION}_amd64$" sha256sums | sha256sum -c \
   && rm sha256sums \
   && mv dumb-init_${DUMB_INIT_VERSION}_amd64 /usr/bin/dumb-init \
   && chmod +x /usr/bin/dumb-init \
-  && apk del .install-deps
+  && python3 -m pip install argparse jinja2 kubernetes urllib3 aionotify==0.2.0 \
```

Comment:
Why don't you create a *python requirements* file to avoid such lists of packages?